### ISSUE TITLE
A-10C Radio Panel Issue

### ISSUE THREAD

**Issue by CrypticVillain** [DCS-Skunkworks/DCSFlightpanels#89]
Generic Radio panel does not work. Similar issues as the F-18C, where certain buttons and dials don't work and most importantly the radio toggles on the left hand side of the panel don't work at all. So you can't switch or select from comm 1 or 2 or nav 1 etc etc

**Comment by WarLord211** [comment]
I tested A-10C it works more or less. 
every time I turn the knob to change the freq the displays flicker. Have to hit the ACT/STBY button offten to get an change of the freq. Also jumps between presets and norm freq numbers as well
tested with last master branch of BIOS and latest DCSFP (10.07.2020)

in the Tomcat i noticed that sometimes i had to press the ACT/STBY button twice to get an change.

**Comment by jdahlblom** [comment]
OK, so the generic radio panel is broken. Not the pre-programmed radio panel for A-10C?

**Comment by CrypticVillain** [comment]
sorry meant the pre-programmed radio panel. The one that has all the bindings set for you and you cant edit.

**Comment by WarLord211** [comment]
i mean also the pre programmed

**Comment by jdahlblom** [comment]
Just tested, works flawlessly?
Are we talking official DCS or open beta?
Windows machine updated properly and .NET Framework 4.7 installed?

**Comment by WarLord211** [comment]
on my side Win10 version 2004; all updates; DCS is open beta, BIOS FP fork last master branch

**Comment by jdahlblom** [comment]
OK thx, let's see what CrypticVillain is using too.

**Comment by CrypticVillain** [comment]
I'm using windows 10 1903, Lastest DCS open beta, lastest dcs bios, lastest dcs flight panel, and .net framework 4.8
The 2004 windows update will not auto update till june 16th. So if that is a requirement for dcs bios to function properly then a lot of people will be having issues.

**Comment by WarLord211** [comment]
no BIOS dont interest the Windows version. which BIOS you use?

latest release (recomended for stable)
or Master branch (recomended for Open beta
Hub (from my standpoint not recommended, because outdated)

**Comment by CrypticVillain** [comment]
I'm using this version of bios https://github.com/DCSFlightpanels/dcs-bios/releases/tag/0.7.35

**Comment by jdahlblom** [comment]
Warlord and me confirmed it isn't working properly if you already have had a profile open and then change to e.g. A-10C. The radio is completely erratic.
If you open DCSFP and have it open the A-10C profile directly, do you have the same problems then?

**Comment by CrypticVillain** [comment]
That did the trick! No more issues with the profile assuming you close fp and reopen it without a profile loaded, then open the profile you want. This also fixed the issues I was having with every other radio profile even the ones I had made. The one exception to this is the F-18 non generic radio panel. You still can not switch presents using the large dial. This has been an issue for a long time. Other then that thank you for figuring out a work around! Hopefully this issue is fixed properly though! I appreciate all the effort you guys put into this! I've donated in the past and will donate again!
